src/QMCWaveFunctions/tests/test_MO_spinor.cpp

```diff
@@ -78,7 +79,7 @@ void test_lcao_spinor()
 
   Libxml2Document doc;
   bool okay = doc.parseFromString(particles);
-  REQUIRE(okay);
+  CHECK(okay);
```

Comment:
I think it is better to have REQUIRE here. If parsing input goes wrong, all the rest will just produce garbage. So better to stop here.

Reply:
Agreed, I overlooked these when I replaced all the REQUIRES with CHECKS